frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepository.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { Formik } from 'formik';
+import { safeLoad } from 'js-yaml';
+import { useTranslation } from 'react-i18next';
+import { k8sCreateResource } from '@console/dynamic-plugin-sdk/src/utils/k8s';
+import { history } from '@console/internal/components/utils';
+import { EditorType } from '@console/shared/src/components/synced-editor/editor-toggle';
+import { safeJSToYAML } from '@console/shared/src/utils/yaml';
+import { ProjectHelmChartRepositoryType } from 'packages/helm-plugin/src/types/helm-types';
+import { ProjectHelmChartRepositoryModel } from '../../../models';
+import CreateProjectHelmChartRepositoryForm from './CreateProjectHelmChartRepositoryForm';
+import {
+  convertToForm,
+  convertToProjectHelmChartRepository,
+} from './projecthelmchartrepository-create-utils';
+import { validationSchema } from './projecthelmchartrepository-validation-utils';
+
+export interface CreateProjectHelmChartRepositoryProps {
+  resource: ProjectHelmChartRepositoryType;
+  namespace: string;
+}
+
+const CreateProjectHelmChartRepository: React.FC<CreateProjectHelmChartRepositoryProps> = ({
+  resource,
+  namespace,
+}) => {
+  const { t } = useTranslation();
+
+  const initialValues = React.useRef({
+    editorType: EditorType.Form,
+    yamlData: safeJSToYAML(resource, 'yamlData', {
+      skipInvalid: true,
+    }),
+    formData: convertToForm(resource),
+    formReloadCount: 0,
```

Comment:
Do we need this? I don't see a reload button in the form.

Reply:
Yeah..removed